crypto/idea/idea_local.h

```diff
@@ -9,12 +9,10 @@
 
 #define idea_mul(r,a,b,ul) \
 ul=(unsigned long)a*b; \
-if (ul != 0) \
-        { \
+if (ul != 0) { \
         r=(ul&0xffff)-(ul>>16); \
         r-=((r)>>16); \
-        } \
-else \
+} else \
         r=(-(int)a-b+1);        /* assuming a or b is 0 and in range */
```

Comment:
Our coding style says that if the "true" body of an if statement is a braced block, then the "false" part must as well.  In other words:

```suggestion
} else { \
        r=(-(int)a-b+1);        /* assuming a or b is 0 and in range */ \
}
```

Reply:
Right, that's probably one of the differences between the [OpenSSL coding style](https://www.openssl.org/policies/technical/coding-style.html) and the [Linux kernel coding style](https://www.kernel.org/doc/Documentation/process/coding-style.rst).

I was hoping I could introduce the [checkpatch.pl](https://docs.kernel.org/dev-tools/checkpatch.html) script as a CI job, once I've fixed most current style issues in the OpenSSL code base.. It looks like this might not be straightforward.

Reply:
The block wasn't braced prior to running `checkpatch.pl`. Since it's not `checkpatch.pl` removing the braces, the additional OpenSSL rule will probably not be an issue.

Rebased and added the missing braces.